lib/matplotlib/rcsetup.py

```diff
@@ -328,6 +338,20 @@ def validate_font_properties(s):
     'verbose',
     ['silent', 'helpful', 'debug', 'debug-annoying'])
 
+def validate_whiskers(s):
+    if s=='range':
+        return 'range'
+    else:
+        try:
+            v = validate_nseq_float(2)(s)
+            return v
+        except:
+            try:
+                v = float(s)
+                return v
+            except:
+                raise ValueError('not a float')
```

Comment:
This should say "not a valid wisker value ['range', float, (float, float)]" or something to that effect.